jsonschema_lexer/data/keywords/draft-01.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
No one uses anything before draft 3 (arguably no one uses draft 3 either). We don't even need to bother with it.

Reply:
Meaning draft 1 and 2 -- leave 3, it still occasionally pops up unfortunately.

Reply:
Okay @Julian, then should I remove those files ?

Reply:
Yes please!